test/unit/memory_spec.lua

```diff
@@ -0,0 +1,35 @@
+local helpers = require("test.unit.helpers")
+
+local cimport = helpers.cimport
+local cstr = helpers.cstr
+local eq = helpers.eq
+local ffi = helpers.ffi
+local to_cstr = helpers.to_cstr
+
+local cimp = cimport('stdlib.h', './src/nvim/memory.h')
+
+describe('xstrlcat()', function()
+  local test_xstrlcat = function(dst, src, dsize)
```

Comment:
Is not current convention “use `local function name(args)`”?

// BTW, anybody knows how to make GH use “Add single comment” on `<C-Enter>`?